content/blog/2025/oddcontrast-guide.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Make the Web a More a Colorful Place!
+sub: A guide to using new color spaces & formats with OddContrast
+author: oddbird
+date: 2025-06-05
+tags:
+  - Article
+  - Accessibility
+  - Color
+  - Design
+  - OddContrast
+  - OddTools
+image:
+  src: blog/2025/oddcontrast-guide.jpg
+  alt: >
+    OddContrast displays P3 gamut range
+    in Oklch color format
+summary: |
+  OddBird's color tool not only checks contrast ratios,
+  but supports the new CSS color formats and spaces.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+For years designers and developers were limited to colors in the sRGB colors
+space, using formats like Hexadecimal, RGB, and HSL. As display technology
```

Comment:
```suggestion
space, using formats like hexadecimal, RGB, and HSL. As display technology
```

Reply:
dang it, that was one of the last minute changes I made. I couldn't decide and I went the wrong way :)

Reply:
It's not a big deal -- I think I prefer `Hex` and `hexidecimal` 😄